qemu/tests/cfg/hotplug_mem_stress.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_mem_stress:
+    type = hotplug_mem_stress
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
+    no Host_RHEL.6
+    no RHEL.5
+    no Windows
```

Comment:
I saw the there were unplug steps in this scenario, should we also include rhel 7 host ?

Reply:
Added no rhel6 and no rhel7 guest restriction. Thanks.